dlt/helpers/dbt/profiles.yml

```diff
@@ -171,3 +171,16 @@ databricks:
       http_path: "{{ env_var('DLT__CREDENTIALS__HTTP_PATH') }}"
       token: "{{ env_var('DLT__CREDENTIALS__ACCESS_TOKEN') }}"
       threads: 4
+
+
+clickhouse:
```

Comment:
I have disabled the dbt tests for clickhouse, the connection set up here works, but since we have the table prefixes after running the seed on jaffle for example the tests will fail because the tables are not found. generally I think it will work though.